proto/proto/penumbra/view/v1alpha1/view.proto

```diff
@@ -46,6 +47,8 @@ service ViewProtocolService {
 
   // Query for a note by its note commitment, optionally waiting until the note is detected.
   rpc NoteByCommitment(NoteByCommitmentRequest) returns (NoteByCommitmentResponse);
+  // Query for a swap by its swap commitment, optionally waiting until the swap is detected.
+  rpc SwapByCommitment(SwapByCommitmentRequest) returns (SwapByCommitmentResponse);
```

Comment:
Since swaps are different from notes now (we're not wrapping a swap commitment in a note, as with the former swap nft), we also need to be able to query the view service for swaps.  However, unlike notes, we probably don't need a flexible query, since we care about this for the purpose of looking up data on a swap we submitted and want to claim.